### PR TITLE
Add color-adjust declaration support

### PR DESCRIPTION
**Pull request by soul-wish**
Here we go in fixing #1001 issue. @ai, can you check this PR, please? Because I'm not sure about some modifications I've made. Haven't found documentation or contribution guide for Autoprefixer, unfortunately. What do you think about adding such type of documentation/guide in the future? Even simple `CONTRIBUTING.md` file would be helpful for newcomers.
I'll try to leave some comments in my commit below ;)

### CODE REVIEW

File: data/prefixes.js [hunk 1]
```diff
@@ -779,3 +779,12 @@ f(over, { match: /a #1/ }, browsers =>
         browsers
     })
 );
+
+// CSS color-adjust property
+f(require('caniuse-lite/data/features/css-color-adjust.json'), browsers =>
+    prefix(['color-adjust'], {
+        mistakes: ['-khtml-', '-ms-', '-moz-', '-o-'],
```
Comment (soul-wish) [inline data/prefixes.js]: If I got it in the correct way, this array prevents Autoprefixer from adding unnecessary prefixes. In our case there is no need to add any prefixes except `-webkit-`.

File: test/autoprefixer.test.js [hunk 2]
```diff
@@ -144,7 +144,7 @@ const COMMONS = [
     'advanced-filter', 'element', 'image-set', 'image-rendering',
     'mask-border', 'writing-mode', 'cross-fade', 'gradient-fix',
     'text-emphasis-position', 'grid', 'grid-area', 'grid-template',
-    'grid-template-areas'
+    'grid-template-areas', 'color-adjust'
```
Comment (soul-wish) [inline test/autoprefixer.test.js]: Should I add 'color-adjust' declaration to the array of `COMMONS`? Not sure about this change. It was a little hard to get the whole structure and logic inside this test file.

File: test/autoprefixer.test.js [hunk 3]
```diff
@@ -422,6 +422,7 @@ describe('hacks', () => {
     it('supports text-decoration',      () => test('text-decoration'));
     it('ignores modern direction',      () => test('animation'));
     it('supports overscroll-behavior',  () => test('overscroll-behavior'));
+    it('supports webkit prefixes',      () => test('color-adjust'));
```
Comment (soul-wish) [inline test/autoprefixer.test.js]: Is the name of test is ok? I tried to get the correct naming logic from the test cases above.

Comment (ai) [reply test/autoprefixer.test.js]: Nope, the test name is incorrect. Check others, they contain hack name, not prefix.

File: test/autoprefixer.test.js (hunk 2 above)
Comment (ai) [reply test/autoprefixer.test.js]: Yes, you should add it to `COMMONS` to run `color-adjust` with not-duplication and prefixes removing tests

File: data/prefixes.js (hunk 1 above)
Comment (ai) [reply data/prefixes.js]: Not, it is not correct. Check other prefixes, they don’t have `mistakes`. `mistakes` is only for popular mistakes. I think nobody wrote `-khtml-color-adjust` in the real world.

**Comment by soul-wish** [comment]
@ai thanks for all your help! I've updated PR according to your instructions. Let me know if I've missed something.

**Comment by soul-wish** [comment]
@ai, @yuriyalekseyev found some issues in original PR, so there is updated version. 😉

**Comment by yuriyalekseyev** [comment]
@soul-wish @ai  it seems that tests are green with a new version of caniuse 👍

**Comment by soul-wish** [comment]
Now we have issue only with size-limit package and increased package size. I hope @ai will tell us what do do with this issue (not sure we need to modify size-limit configuration inside our PR).

**Pull request merged**